grails-app/services/org/pih/warehouse/core/LocationService.groovy

```diff
@@ -89,11 +89,15 @@ class LocationService {
 		return locations
 	}
 
-	def getLocations(String [] fields, Map params, Boolean isSuperuser, String direction, Location currentLocation) {
+	def getLocations(String [] fields, Map params, Boolean isSuperuser, String direction, Location currentLocation, User user) {
 
 		def locations = new HashSet()
 		locations += getLocations(fields, params)
 
+		if (params.locationChooser) {
```

Comment:
I've noticed this a few times and wanted to make a constructive comment related to your variable / parameter naming. In general, you want to name variables and parameters in a way that describes what they are doing, rather than where / how they are applied. 

In this case, it would be preferable to use a name like applyUserFilter or removeInaccessibleLocations that describes what the ensuing code is doing. 

This improves the code in a few ways: 
* it becomes self-documenting
* it allows us to apply the same filtering in non-Location Chooser context without having to rename the variable / parameter or add a new variable / parameter.